sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -217,4 +219,35 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
       .where("location.id = :locationId", { locationId })
       .getRawOne();
   }
+
+  /**
+   * Get an institution's locations with designation statuses.
+   * @param institutionId institution id
+   * @returns Institution Locations with designation statuses.
+   */
+  async getInstitutionLocationsWithDesignationStatus(
```

Comment:
The method `getInstitutionLocationsWithDesignationStatus` seems to be eliminating the need of `getLocation` above. Can we keep just one of them maybe?
@ann-aot FYI

Reply:
In `getLocation` the where clause is `locationId`, its is `insitutionId`
![image](https://user-images.githubusercontent.com/77353155/160493939-b2d9c048-6d53-4b6a-ad36-244fe2a1b3c2.png)